gensim/matutils.py

```diff
@@ -479,6 +479,38 @@ def isbow(vec):
         return False
     return True
 
+def symmetric_kl(distrib_p, distrib_q):
+    return numpy.sum([stats.entropy(distrib_p, distrib_q), stats.entropy(distrib_p, distrib_q)])
+
+def arun_metric(self, min_num_topics=10, max_num_topics=50, iterations=10):
+    """
+    Implements Arun metric to estimate the optimal number of topics:
+    Arun, R., V. Suresh, C. V. Madhavan, and M. N. Murthy
+    On finding the natural number of topics with latent dirichlet allocation: Some observations.
+    In PAKDD (2010), pp. 391–402.
+    
+    :param min_num_topics: Minimum number of topics to test
+    :param max_num_topics: Maximum number of topics to test
+    :param iterations: Number of iterations per value of k
+    :return: A list of len (max_num_topics - min_num_topics) with the average symmetric KL divergence for each k
```

Comment:
Does `length of (max - min)` mean one of range boundaries is exclusive? Which one?